tree-sitter-python/test/Main.hs

```diff
@@ -31,5 +33,6 @@ prop_simpleExamples = property $ do
   "pass" `shouldParseInto` Py.Module { Py.ann = (), Py.extraChildren = [Right pass] }
   "1" `shouldParseInto` Py.Module { Py.ann = (), Py.extraChildren = [Right one] }
   "expensive" `shouldParseInto` Py.Module { Py.ann = (), Py.extraChildren = [Right function] }
+  "1\npass" `shouldParseInto` Py.Module { Py.ann = (), Py.extraChildren = [Right one, Right pass] }
```

Comment:
This is a pretty straightforward test of #195. We didn’t see this in manual testing because we were testing against `"pass\npass"`, effectively a palindrome 😂